development/generate_credits.py

```diff
@@ -26,7 +26,9 @@
 
 sections = ['UH-Team_New', 'UH-Team_Old', 'Patchers', 'Translators', 'Packagers', 'Special Thanks']
 section_widgets = {s: 'credits_' + s.lower() for s in sections}
-section_widgets.update({'UH-Team-2016/2017': 'credits_team_2016','UH-Team-2015': 'credits_team_2015', 'Special Thanks': 'credits_thanks'})
+section_widgets.update({'UH-Team-2016/2017': 'credits_team_2016',
+						'UH-Team-2015': 'credits_team_2015',
+						'Special Thanks': 'credits_thanks'})
```

Comment:
Yes, I use a tabwide of 4 char, but github show 8 char for a tab, so that´s look ugly in github.

I would suggest, with one tab:
```
section_widgets.update({
         'UH-Team-2016/2017': 'credits_team_2016',
         'UH-Team-2015': 'credits_team_2015',
         'Special Thanks': 'credits_thanks'})
```